assets/css/widescreen.css

```diff
@@ -1,16 +1,638 @@
+    background-color: #ff6b00;
+    border: 2px solid #ff6b00;
+    color: #fff;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    background-color: #1c1a19;
+    padding: 0 142px 0 142px;
+    background-image: url('../img/Frame.png');
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-position-y: 200px;
+  }
+
+  .about-me .intro {
+    height: 652px;
+  }
+
+  .about-me h1 {
+    padding-top: 120px;
+    color: #ff6b00;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 88px;
+    letter-spacing: 0.37px;
+    margin-bottom: 0;
+  }
+
+  .about-me .desc {
+    width: 566px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .about-me button.btn-get-my-resume {
+    width: 200px;
+    height: 9vh;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 2vh;
+    line-height: 24px;
+    padding: 12px;
+    color: #fff;
+    background-color: #ff6b00;
+  }
+
+  .about-me button.btn-get-my-resume:hover {
+    background: #fff5e1;
+    border: 2px solid #ff6b00;
+    color: #ff6b00;
+  }
+
+  .about-me button.btn-get-my-resume:focus {
+    background-color: #ff6b00;
+    border: 2px solid #ff6b00;
+    color: #fff;
+  }
+
+  hr.about-me-line {
+    flex: none;
+    border: 1px solid #fff;
+    flex-grow: 0;
+    margin: 10px 0;
+  }
+
+  .tech-stacks-and-skils {
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    padding-top: 120px;
+  }
+
+  .tech-stacks-and-skils h1 {
+    margin: 0;
+    padding: 0;
+    color: #fff;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+  }
+
+  .language,
+  .frameworks,
+  .skils {
+    margin-right: 176px;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  .language-title,
+  .frameworks-title,
+  .skils-title {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+
+  .stack-icon {
+    width: 23px;
+    height: 23px;
+    margin-right: 23px;
+  }
+
+  .language,
+  .frameworks,
+  .skils,
+  ul {
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+  }
+
+  .language,
+  .frameworks,
+  .skils,
+  ul li {
+    list-style-type: none;
+    color: #fff;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 22px;
+    line-height: 50px;
+  }
+
+  .get-in-touch {
+    padding: 130px 142px 0 142px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .get-in-touch .get-in-touch-form {
+    width: 684px;
+  }
+
+  .get-in-touch .get-in-touch-desc {
+    width: 455px;
+    margin-right: 24px;
+    padding-top: 0;
+    color: #172b4d;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .get-in-touch .input-control {
+    width: 100%;
+  }
```

Comment:
- [ ] The final section of the website isn't properly centered, please make sure to address this issue.


| Design  |
| ------------- |
| ![](https://i.imgur.com/BWvzv7U.png)  |

Reply:
Thanks i will take  them into account.

Reply:
https://user-images.githubusercontent.com/69638013/155669828-64fc9067-06e1-4472-b3fa-823634fa9e6c.mp4

This is how your website looks at 992px and 1080px,  it is important you specify the right width size